main.go

```diff
@@ -14,7 +14,6 @@ import (
 	witmiddleware "github.com/fabric8-services/fabric8-wit/goamiddleware"
 	"github.com/fabric8-services/fabric8-wit/log"
 	"github.com/goadesign/goa"
-	"github.com/goadesign/goa/logging/logrus"
```

Comment:
why remove it in the import but not line 40?

Reply:
not sure why it was removed, but it's been restored with  `goalogrus` alias now.